src/chainparams.cpp

```diff
@@ -97,7 +97,7 @@ class CMainParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_SEGWIT].nTimeout = 1510704000; // November 15th, 2017.
 
         // Deployment of SEGSIGNAL
-        consensus.vDeployments[Consensus::DEPLOYMENT_SEGSIGNAL].bit = 2;
+        consensus.vDeployments[Consensus::DEPLOYMENT_SEGSIGNAL].bit = 4;
```

Comment:
This was chosen because it was what was listed in the NYC agreement, I made a separate BIP that uses bit 2 however.